resources/aws/environmentDefaults.go

```diff
@@ -157,7 +157,7 @@ func agentSandboxDefault() environmentDefault {
 				defaultFakeintakeLBs: []FakeintakeLBConfig{
 					{listenerArn: "arn:aws:elasticloadbalancing:us-east-1:376334461865:listener/app/fakeintake/3bbebae6506eb8cb/eea87c947a30f106", baseHost: ".lb1.fi.sandbox.dda-testing.com"},
 					{listenerArn: "arn:aws:elasticloadbalancing:us-east-1:376334461865:listener/app/fakeintake2/e514320b44979d84/3df6c797d971c13b", baseHost: ".lb2.fi.sandbox.dda-testing.com"},
-					{listenerArn: "arn:aws:elasticloadbalancing:us-east-1:376334461865:listener/app/fakeintake3/1af15fb150ca4eb4/88e1d12c35e7aba0", baseHost: ".lb3.fi.sandbox.dda-testing.com"},
+					{listenerArn: "arn:aws:elasticloadbalancing:us-east-1:376334461865:listener/app/fakeintake3/1af15fb150ca4eb4/041c6a59952354c1", baseHost: ".lb3.fi.sandbox.dda-testing.com"},
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
How was it deleted? Clean up job or manually?

Reply:
Probably manually. Anybody could have deleted it